packages/pools/src/constants/pools/56.ts

```diff
@@ -13,6 +13,15 @@ export const livePools: SerializedPool[] = [
     tokenPerBlock: '10',
     isFinished: false,
   },
+  {
+    sousId: 360,
+    stakingToken: bscTokens.cake,
+    earningToken: bscTokens.axl,
+    contractAddress: '0x32f0414a4e970a3edd9e7db367A43348F25D74cD',
+    poolCategory: PoolCategory.CORE,
+    tokenPerBlock: '0.05358',
```

Comment:
I think this is wrong. contract show `53580`
53580 / 1e18 = 0.000000000000053580